### PR TITLE
Add BC Sans font, apply to style.css, increment version

### PR DESCRIPTION
**Pull request by michaelpnelson**
This adds the BC Sans web font to the assets of new SMK apps (both default and mobile) and updates style.css to set BC Sans as the font family.

This also updates the version as well as the SMK dependency version, to consume an SMK release that better supports BC Sans styling.

### CODE REVIEW

File: smk-create/template-default/resources/assets/style.css [hunk 1]
```diff
@@ -12,7 +17,11 @@ body {
     justify-content: stretch;
     align-items: stretch;
 
-    font-family: Helvetica,Arial,sans-serif;
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
+}
+
+button, input {
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
```
Comment (dgboss) [inline smk-create/template-default/resources/assets/style.css:24]: Consider setting the font-family for buttons to 'inherit' in order to make updating the app wide font a little easier for users.

font-family: inherit

**Review by dgboss — APPROVED** [review]
The code looks good, though I still don't think the font is appropriate/adequately legible for web use!

File: smk-create/template-default/resources/assets/style.css (hunk 1 above)
Comment (michaelpnelson) [reply smk-create/template-default/resources/assets/style.css:24]: Done.

File: smk-create/template-mobile/resources/assets/style.css [hunk 2]
```diff
@@ -12,7 +17,11 @@ body {
     justify-content: stretch;
     align-items: stretch;
 
-    font-family: Helvetica,Arial,sans-serif;
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
+}
+
+button, input {
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
```
[the remark this answers is not in the document]
Comment (michaelpnelson) [reply smk-create/template-mobile/resources/assets/style.css:24]: Done.

**Pull request merged**